index.html

```diff
@@ -0,0 +1,208 @@
+          <a href="#/"><i class="fa fa-twitter white pb-2"></i></a>
+          <span class="white header-text">English</span>
+          <span class="white header-text">My Page</span>
+          <span class="white header-text">Logout</span>
+        </div>
+      </div>
+    </div>
+  </header>
+  <nav>
+    <div class="container-fluid position-fixed nav-container">
+      <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+      <div class="row align-items-center">
+          <div class="col-12 col-md-5 d-flex justify-content-center align-items-center pt-md-0 pb-md-0">
+            <a href="index.html" class="d-flex justify-content-center" ><img src="img/logo.png" class="nav-icon pt-md-0"></a>
+          </div>
+          <div class="col col-md-4 d-flex align-items-center fw-bold">
+            <ul class="nav-links mb-0">
+              <a href="about.html"><li>About</li></a>
+              <a href=""><li>Program</li></a>
+              <a href=""><li>Join</li></a>
+              <a href=""><li>Sponsor</li></a>
+              <a href=""><li>News</li></a>
+            </ul>
+          </div>
+          <div class="col-md-3 align-items-center justify-content-start d-none d-md-flex ps-5">
+            <p class="p-1 m-1 campaign-text">CC Campaign</p>
+          </div>
+      </div>
+    </div>
+  </nav>
+  <section>
+    <div class="container-fluid ps-md-5" id="intro">
+      <div class="row ps-md-5 ms-md-5 justify-content-start">
+        <div class="col-12 col-md-7 ms-md-5 mt-5 pt-5">
+          <h1 class="fs-5 mb-1 intro-hello">"WE ARE NOW IN MYANMAR!"</h1>
+          <h1 class="intro-title mb-4">SUPREME SUMMIT TO CHANGE YOUR LIFE 2023</h1>
+          <p class="p-2 border-4 border border-white mt-4 intro-desc">Reach Your Ultimate Human Potential And Many World Class Speakers.Discover How You Can Live Your Passion, And Build Your Income... Even If You've Repeatedly Failed Before</p>
+          <p class="intro-date fs-4 pt-4 mb-0">2023.10.15(MON) ~ 19(FRI)</p>
+          <p class="intro-place pb-5 mb-0">@ National Museum of Yangon</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section>
+    <main>
+      <div class="container-fluid project-container">
+        <div class="d-flex flex-column justify-content-center align-items-center pt-4 pb-4">
+          <h2 class="mb-2 fs-5">Main Program</h2>
+          <div class="orange-line"></div>
+        </div>
+        <div class="row justify-content-center gap-md-0 pt-md-4">
+          <div class="col-11 col-md-2 d-flex flex-md-column gap-md-1 justify-content-md-start align-items-center justify-content-between project-sec mb-1 me-md-1 m-md-0">
+            <div class="row align-items-center p-2">
+              <div class="col-2 col-md-12 d-md-flex justify-content-md-center p-1 p-md-2">
+                <img src="img/brain.png" class="program-icon">
```

Comment:
- [ ] please like the previous reviewer noted, Kindly ensure all images on your project have a descriptive alt attribute. it helps describes the image or at least states what the image is trying to convey to the person seeing the page. please do that to all images in this section.